src/api/PluginManager.ts

```diff
@@ -81,3 +93,15 @@ export function startCorePlugins() {
         }
     }
 }
+
+function loadPluginBundle(plugin: string) { 
+    const bundleZip = new ZipFile(PLUGINS_DIRECTORY + `${plugin}.zip`, 0, "r");
```

Comment:
if you pass the zip as argument instead, you don't have to open two zips during the normal load workflow